sample/kotlin/src/main/kotlin/com/datadog/android/sample/webview/WebViewModel.kt

```diff
@@ -16,7 +17,10 @@ internal class WebViewModel(
 
     fun onResume() {
         localServer.start(
-            "https://datadoghq.dev/browser-sdk-test-playground/webview-support/"
+            "https://datadoghq.dev/browser-sdk-test-playground/" +
+                "?client_token=${BuildConfig.DD_CLIENT_TOKEN}" +
+                "&application_id=${BuildConfig.DD_RUM_APPLICATION_ID}" +
+                "&site=datadoghq.com"
```

Comment:
should we have here a site used by the app/SDK? the one from `BuildConfig.DD_SITE_NAME`